web-api/src/persistence/elasticsearch/getCasesByFilters.ts

```diff
@@ -112,15 +126,15 @@ export const getCasesByFilters = async ({
         'highPriority.BOOL': true,
       },
     };
-    filters.push(procedureTypeFilter);
+    mustClause.push(procedureTypeFilter);
   }
 
   const searchResults = await applicationContext.getSearchClient().search({
     _source: source,
     body: {
       query: {
         bool: {
-          must: filters,
+          must: mustClause,
```

Comment:
this might be better suited to `filter` instead of `must`. I think they're effectively the same, it's just that `filter` does not go through the added work of scoring and is thus more performant.